build.gradle

```diff
@@ -24,6 +24,7 @@ allprojects {
   apply plugin: 'java-library'
 
   repositories {
+    mavenLocal()
```

Comment:
Should this go into `maru.kotlin-common-minimal-conventions.gradle`?

Reply:
I was planning to remove this before any code is merged in. Just a temporary until we have a Besu release out with the changes I need.